### PR TITLE
refactor: remove unneeded .stackblitzrc, update template dependencies

### PR DESCRIPTION
**Pull request by fvsch**
This PR harmonizes what basic config files we find in a starter template.

### What's in `package.json`

Almost all starters have a `package.json` file. I've tried to harmonize those files to follow this pattern:

```json
{
  "name": "{folder-name}-demo",
  "version": "0.0.0",
  "private": true,
  "scripts": { … },
  "dependencies": { … },
  "devDependencies": { … }
}
```

Some templates may not have `"scripts"` (most for some EngineBlock starters), some may not have `"devDependencies"`.

Open questions:

1. What should we use for the package name?
    - Should we include one at all?
    - Is the `"{folder-name}-demo"` pattern okay, or do we want to use “starter” or “template” instead of “demo” maybe?
    - Note that if we end up using a name that matches an existing package on npm, running `npm info` will show information for that package merged with info from the local `package.json`, which is quite strange, but maybe a very rare problem.
2. Is the `"version"` even useful? I’m tempted to just drop it.

### Removing all the `.stackblitzrc` files

All the `.stackblitzrc` files in WebContainer projects were spelling out defaults, such as `{ "installDependencies": true, startCommand: "npm run dev" }`.

I removed them for the sake of readability, but if we want to include one with defaults in every WC starter template we could do that too.

### Making sure every WC template has a `.gitignore`

For every WebContainer-based template, I made sure it had a `.gitignore` file that listed at least:

- `node_modules`
- whatever caching folder the framework is using (`.next`, `.nuxt`, `.svelte-kit`, etc.)
- whatever build output folder resulted from running `npm run build` (`build`, `dist`, etc.)

### Harmonizing `index.html` files

- Made sure every one had a `<!DOCTYPE html>`
- Removed `<meta http-equiv="X-UA-Compatible" content="IE=edge">`
- Used the same minimal viewport meta everywhere: `<meta name="viewport" content="width=device-width">`

### Harmonizing base CSS styles

We had base `style.css` files which looked like:

```css
h1, h2 {
  font-family: Lato;
}
```

That's not super useful, especially since the preview documents don't load the Lato fonts at all. (They used to, in EngineBlock projects, but we removed those fonts in early 2022 for performance reasons.)

Instead I'm proposing a very basic stylesheet looking like this:

```css
* {
  box-sizing: border-box;
}

body {
  margin: 0;
  padding: 1rem;
  font-family: system-ui, sans-serif;
  color: black;
  background-color: white;
}

h1 {
  font-weight: 800;
  font-size: 1.5rem;
}
```

### CODE REVIEW

File: bootstrap-5/package.json [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-  "name": "node-starter",
+  "name": "bootstrap-5-demo",
   "version": "0.0.0",
+  "private": true,
   "scripts": {
-    "start": "webpack serve"
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
   },
```
Comment (fvsch) [inline bootstrap-5/package.json:9]: Converted the Bootstrap 5 example from webpack to Vite.
Simpler less verbose config, and HMR for free.

File: express-simple/README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
-# node-dmbz1u
+# Simple express demo
 
-[Edit on StackBlitz ⚡️](https://stackblitz.com/edit/node-dmbz1u)
\ No newline at end of file
+> Express: fast, unopinionated, minimalist web framework for Node.js
+
+This project shows a simple express server serving a single HTML page and using `express.static` to serve static files.
+
+Check out the [express documentation](https://expressjs.com/) for more information.
```
Comment (fvsch) [inline express-simple/README.md:7]: Apparently we had just saved an automatically generated `README.md` pushed to GitHub by the StackBlitz editor.

File: gsap-react/package.json [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
+    "react-dom": "^18.2.0"
+  },
   "devDependencies": {
-    "react-scripts": "latest"
-  }
+    "react-scripts": "^5.0.1"
+  },
+  "browserslist": [
+    "defaults"
+  ]
```
Comment (fvsch) [inline gsap-react/package.json:21]: The `browserslist` config is required by `react-scripts`.

File: js/tsconfig.json [hunk 4]
```diff
@@ -1,5 +0,0 @@
-{
-  "compilerOptions": {
-    "target": "esnext"
-  }
-}
```
Comment (fvsch) [inline js/tsconfig.json:5]: I’m not sure why the JS EngineBlock starter template would need a `tsconfig.json`. My understanding is that is doesn't.

File: node/package.json [hunk 5]
```diff
@@ -1,7 +1,8 @@
 {
-  "name": "node-starter",
+  "name": "node-demo",
   "version": "0.0.0",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node index.js"
```
Comment (fvsch) [inline node/package.json:6]: Instead of telling users to run `node index.js` in the Terminal as a comment in `index.js`, we can just run it on startup.

**Review by sulco — APPROVED** [review]
I like all the changes very much! I'd drop `version` as well, but I don't have strong feelings about that. 

For the package name, I think the `-starter` makes a bit more sense, keeping it in the same "name space" as we're already using when calling this pieces of StackBlitz experience a "starter".

**Comment by fvsch** [comment]
> I'd drop `version` as well, but I don't have strong feelings about that.

Dropped it. I agree that it's not useful for projects.

> For the package name, I think the `-starter` makes a bit more sense

Changed from `-demo` suffix to `-starter` suffix.

**Pull request merged**